benchmarks/benchmark_serving.py

```diff
@@ -156,16 +157,26 @@ def to_dict(self):
     }
 
 
-def get_tokenizer(model_id: str, tokenizer_name: str) -> Any:
+def get_tokenizer(
+    model_id: str,
+    tokenizer_name: str,
+    use_hf_tokenizer: bool,
+) -> Any:
   """Return a tokenizer or a tokenizer placholder."""
   if tokenizer_name == "test":
+    print("Using test tokenizer")
     return "test"
+  elif use_hf_tokenizer:
+    print(f"Using HuggingFace tokenizer: {tokenizer_name}")
+    return AutoTokenizer.from_pretrained(tokenizer_name)
```

Comment:
User needs to accept the MixtralAI (or other model) agreeement on HuggingFace, else this line will error out. Can you add a comment about it for future users?

Reply:
Sure thing!

Reply:
Added comment accordingly.